package.json

```diff
@@ -61,7 +61,6 @@
     "axios": "0.21.4",
     "ejs": "^3.1.7",
     "handlebars": "4.7.7",
-    "node-notifier": "8.0.1",
-    "eslint-plugin-import/**/json5": "^1.0.2"
```

Comment:
yarn keeps trying to remove this package, should I obey it or remove the changeset from the PR?